src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -5,6 +5,10 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
nit: I know this is added by VS automatically, it's better if could be removed to keep consistency (we don't have it for 1.sql and 2.sql)